src/main/java/org/fenixedu/learning/domain/degree/DegreeSite.java

```diff
@@ -72,4 +90,10 @@ public void delete() {
         return Stream.of(categoryForSlug("announcement")).filter(Objects::nonNull).map(Category::makeWrap);
     }
 
+    private CMSFolder folderForPath(MenuContainer parent, String path) {
```

Comment:
Given that folders are optional in the CMS, it does not make sense to blow up if that particular folder is not found.

One might want not to organize degree sites in folders (or even set them up in a custom, on different path), so I think it would just be better to do nothing in this case.